>From bb6ac51c480a4bff3ebb3377ec583cae22538480 Mon Sep 17 00:00:00 2001
From: Michael Biebl <biebl@debian.org>
Date: Sun, 2 Oct 2016 01:41:34 +0200
Subject: [PATCH] Make sure all files are installed into the debhelper binary
 package

Now that we are building more then one binary package, dh_auto_install
no longer uses debian/debhelper by default. Since the transitional
dh-systemd package is supposed to be empty, override dh_auto_install and
set --destdir=debian/debhelper to make sure all files end up in the
debhelper binary package.
---
 debian/rules | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/debian/rules b/debian/rules
index addfbff..b507359 100755
--- a/debian/rules
+++ b/debian/rules
@@ -16,3 +16,6 @@
 # Disable as they are unneeded (and we can then be sure debhelper
 # builds without needing autotools-dev, dh-strip-nondetermism etc.)
 override_dh_update_autotools_config override_dh_strip_nondeterminism:
+
+override_dh_auto_install:
+	dh_auto_install --destdir=debian/debhelper
-- 
2.9.3

